modules/microshift-automation-example-rpm-systems.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * microshift/microshift-auto-recover-manual-backup.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-automation-example-rpm-systems_{context}"]
+= Automation integration example with systemd for RPM systems
+
+Use the following example to understand how, when the `microshift.service` enters a failed state, `systemd` starts the `microshift-auto-recovery.service` unit, which executes the auto-recovery restore process and restarts {microshift-short}. This automates the auto-recovery process for RPM systems using `systemd`.
```

Comment:
I feel like this sentence is too complex.

Reply:
done